src/wasm-lib/kcl/src/settings/types/mod.rs

```diff
@@ -538,6 +554,7 @@ enableSSAO = false
 
 [settings.modeling]
 defaultUnit = "in"
+cameraProjection = "orthographic"
```

Comment:
Confirming that this is Josh's recommendation for starting in Orthographic mode by default now?

Reply:
Oh no I am **not** setting ortho to the default just now. I think we should do that as a later effort because it would break every test and be a very big change. I should add that to the description. This line just adds the setting to a test of how the settings get parsed.

Reply:
Can't we just make it the default, and than change the user settings for out tests to be perspective again so our tests don't have to change?

Reply:
Ah yeah good call @Irev-Dev

Reply:
I've made orthographic the default, and made our test settings explicitly use perspective.